modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationUtils.java

```diff
@@ -84,15 +83,14 @@ public void updateDeviceOperations(Status status, SyncmlDocument syncmlDocument,
                     operation.setStatus(Operation.Status.ERROR);
                     updateOperations(syncmlDocument.getHeader().getSource().getLocURI(), pendingDataOperations);
                     try {
-                        NotificationManagementService service =
-                                WindowsAPIUtils.getNotificationManagementService();
+                        NotificationManagementService nmService = WindowsAPIUtils.getNotificationManagementService();
                         Notification lockResetNotification = new Notification();
                         lockResetNotification.setOperationId(status.getCommandReference());
                         lockResetNotification.setStatus(String.valueOf(Notification.Status.NEW));
                         lockResetNotification.setDeviceIdentifier(deviceIdentifier);
                         lockResetNotification.setDescription(
                                 Constants.SyncMLResponseCodes.LOCKRESET_NOTIFICATION);
-                        service.addNotification(lockResetNotification);
+                        nmService.addNotification(lockResetNotification);
                     } catch (NotificationManagementException e) {
                         throw new WindowsOperationException("Failure occurred in getting notification service", e);
```

Comment:
wrong message passing, this should be some error occured during adding notification.